ui/main/src/app/services/translation/TranslationService.ts

```
@@ -47,45 +53,33 @@ export class I18nService {
-        opfab.multiSelect.noOptionsText = this.translationService.getTranslation('multiSelect.noOptionsText');
-        opfab.multiSelect.noSearchResultsText = this.translationService.getTranslation(
-            'multiSelect.noSearchResultsText'
-        );
-    }
-
     public static setTranslationForRichTextEditor() {
         const root = document.documentElement;
         root.style.setProperty(
             '--opfab-richtext-link-enter',
-            '"' + this.translationService.getTranslation('userCard.richTextEditor.enterLink') + '"'
+            '"' + this.translationLib.getTranslation('userCard.richTextEditor.enterLink') + '"'
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)